### DISCUSSION TITLE
Schema Verification

### DISCUSSION THREAD

**Discussion by AkkafaOguz** [Kong/insomnia#6507] [Help]
Dear Authorized, 

I'm a member of a global team that tries to switch from Postman to Insomnia for API testing. We have an issue to test the schema of response bodies on Insomnia. We've searched but not been able to find any solution or documentation on the web. As a last chance, we want to ask you to be sure that Insomnia has a feature to check schema like postman as you can see at the file that I've attached or not. 

Kind regards,

Oguzhan A.
![Schema](https://github.com/Kong/insomnia/assets/88987698/71613e68-c518-4f0b-b94f-47960b6deec4)

**Comment by mirket999** [comment]
As I experienced and see from discussions, there is no built-in solution for schema validation in Insomnia. They use Chai library but some functionalities are not working yet. Only way is to develop a method in JavaScript that iterates over your schema and does validations. This guy has a contribution but not sure if it will be a solution: https://github.com/Kong/insomnia/pull/6532

**Comment by subnetmarco** [comment]
Working on this as we speak, duplicate of https://github.com/Kong/insomnia/discussions/6268 so closing here.
